### ISSUE TITLE
Image links to exhibits have no discernable texts

### ISSUE THREAD

**Issue by Otterfan** [BCDigSchol/exhibits-omeka-theme#28]
Image links to exhibits on all pages fail the ["Links must have discernible text" inspection](https://dequeuniversity.com/rules/axe/2.2/link-name?application=lighthouse) in Lighthouse.
